gluten-core/src/main/java/io/glutenproject/substrait/rel/LocalFilesNode.java

```diff
@@ -50,7 +50,7 @@ public enum ReadFileFormat {
   }
 
   private ReadFileFormat fileFormat = ReadFileFormat.UnknownFormat;
-  private Boolean iterAsInput = false;
+  private Boolean streamInput = false;
```

Comment:
TBH, the original variable name seems easy to follow. streamInput can be used when Gluten support Flink.

Reply:
fixed.